scapy/layers/l2.py

```diff
@@ -405,7 +405,7 @@ def route(self):
         elif isinstance(fld, IPField):
             return conf.route.route(dst)
         else:
-            return None
+            return None, None, None
```

Comment:
Can you add the corresponding unit test?

Reply:
Done